packages/docs/content/components/button.mdx

```diff
@@ -11,7 +11,13 @@ medicare:
   sketchLink: Vr4yyWw
 ---
 
-import { ArrowIcon, Badge, Button } from '@cmsgov/design-system';
+import { Alert, ArrowIcon, Button } from '@cmsgov/design-system';
+
+<Alert heading="Are you upgrading to v4?">
+
+  [Take a look at our v4 buttons migration guide](/getting-started/migrating-to-new-buttons) to get started.
+  
+</Alert>
```

Comment:
[question] does this version need to change per brand? I guess, does "v4" make sense to someone on mgov who will be on v6? Or someone on hgov that will be on v8?

Reply:
Hmm, that's a good point. The page it points to doesn't change, but this part could be dynamic.